### DISCUSSION TITLE
Asus G14 4900HS

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1573] [Q&A]
I am really very sorry for asking this. but I am not able to understand or find why this version does not support anime matrix. I read about this model not supporting GPU customizations, but can't find anything on anime matrix. Is it possible in the future updates to do so?

Thanks many many times to the developer Sergee and Community!

**Comment by seerge** [comment]
@dev1312000 if your device has anime matrix - you will see it in the app UI, it's supported

**Reply by ghost** [reply]
I definitely have but it does not show. I tried restart and other options. still not available. Take a look.

![firefox_fWl4MEmJrX](https://github.com/seerge/g-helper/assets/76037097/f8117bff-1eea-47b5-9ece-305f0b405ac4)
![AsusMyASUS_NUBheI3Gt9](https://github.com/seerge/g-helper/assets/76037097/dac66dea-2f41-40ab-a879-a37ac0ea5073)

**Reply by seerge** [reply]
And does it show in AC ? :)

**Reply by ghost** [reply]
Yes, it showed in AC. 

my version also doesn't support GPU customizations as I found on reddit?

I just installed it today. that's why too many questions.

**Reply by seerge** [reply]
Does it show now ?

**Reply by ghost** [reply]
No it doesn't show now. see the screenshot. I also don't see it in "extra".

Thanks for quick reply.

**Reply by seerge** [reply]
If it doesn't show in AC now - then it's some hardware / windows problems. You may want to do a hardware reset or delete/re-add this anime-matrix device in windows. 

Good luck

**Reply by ghost** [reply]
Okay I will perform hardware reset. Thank you so much Sergee for active help! Never seen developer being this active.

**Reply by ghost** [reply]
Did hardware reset. Still the same problem. I don't even game on the laptop. only for university software use. What can even cause the problem?

anyway, still I will try to find bug.

**Reply by seerge** [reply]
I don't know :) try to check for matrix device in windows device manager and re-add it

**Reply by ghost** [reply]
Sorry, what do you mean by matrix device?

**Reply by seerge** [reply]
Your anime matix identifies itself just as a USB device in device manager in windows. It needs to be accessible there in a first place in order to let AC / G-Helper to connect to it.

**Reply by ghost** [reply]
have every drivers up-to-date. Requires any special permission?

Desperately trying to get back anime matrix. my little brother loved it.

I read this discussion also.
 Not detecting AniMe Matrix #1066 

But I don't see any errors on my drivers same as him.
![image](https://github.com/seerge/g-helper/assets/76037097/1dc87925-b4f6-47b6-8f64-2748e916d041)

**Reply by seerge** [reply]
I don 't know :) you may check for device id ``0x193B`` with https://www.nirsoft.net/utils/usb_devices_view.html

May be it's broken in your case and not being detected :)

![Screenshot 2023-11-08 123313](https://github.com/seerge/g-helper/assets/5920850/220068fe-37af-4019-9122-4270ea266dfb)

**Reply by seerge** [reply]
I don't know

**Reply by ghost** [reply]
resetted again. found it. but there is nothing in ghelper still.


![image](https://github.com/seerge/g-helper/assets/76037097/4cae0ce8-158a-408b-9cf8-bc56806a6c97)

**Reply by seerge** [reply]
If you don't have a device accessible of course not a single app will be able to do something with your matrix :)

**Reply by ghost** [reply]
how do I make that accessible? in which section should I search that?

I think it can be solved.

**Reply by seerge** [reply]
It's your computer. I don't know what's happening on it.  your problem is not related to g-helper anyhow.
